lib/ansible/modules/cloud/scaleway/scaleway_lb.py

```diff
@@ -70,15 +70,15 @@
     type: bool
     default: 'no'
 
-  wait_timeout:
+  until:
```

Comment:
I don't think this new parameter name is acceptable.
It is confusing because we have a task directive names `until`.
But also it doesn't accept the same content.

The original parameter name is much better, and I would keep it.